.github/workflows/checkstyle.yaml

```diff
@@ -44,6 +44,18 @@ jobs:
           echo "clang format errors:"
           cat clang-format-errors.txt
 
+      - name: prettier
+        # NOTE: Only run prettier on files that differ from master, since
+        # prettier can be slow.
+        run: |
+          git merge-base HEAD origin/master |
+            xargs git diff --name-only --diff-filter=AMRCT |
+            grep -P '\.(js|jsx|ts|tsx|html|css|yaml|json|md|xml)$' |
```

Comment:
I'm sure there are a bunch of yaml, css, html, and md files that don't conform to prettier's style standard.

Should we fix those / see what they look like before enabling?

Reply:
Sure -- https://github.com/buildbuddy-io/buildbuddy/pull/754

Reply:
Seems reasonable if @tylerwilliams  is okay with it.

Reply:
👍